.github/workflows/pull_request_verify.yaml

```diff
@@ -0,0 +1,191 @@
+name: Pull Requests
+
+# When someone comment "/deploy" on anyone PR,
+# this workflow will be triggered and run.
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened, labeled ]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full
+  CKB_CLI_VERSION: v0.40.0  
+jobs:
+#job for checkout the ckb
+  checkout: 
+    name: ckb-checkout  
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+        -  Linux-runner
+        -  mac-runner
+    steps:
+    - name: Set env
+      run: echo "CARGO_TARGET_DIR=$(echo $HOME/target)" >> $GITHUB_ENV
+    - uses: actions/checkout@v2
+      with:
+        submodules: false
+        fetch-depth: 2
+        ref: ${{ github.event.pull_request.head.sha }}
+        repository: ${{github.event.pull_request.head.repo.full_name}}
+        #cargo build 的target，试一下在job 层加 cargo build targe
+    - run: |
+        echo "runner os is ${{ runner.os }}"
+        if [ ${{ runner.os }} == "macOS" ]; then
+        export PATH="/Users/administrator/.cargo/bin:$PATH"
```

Comment:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#adding-a-system-path

Reply:
added the step to set the path